src/styles/index.scss

```diff
@@ -1,3 +1,23 @@
+$main-text-color: #060b35;
+$secondary-text-color: #616070;
+$button-color: #00acdc;
+$button-text-color: #fff;
```

Comment:
Consider moving variables to a separate file and keeping the main file clean (just for imports)